drivers/counter/Kconfig

```
@@ -22,6 +22,12 @@ config COUNTER_SHELL
 	help
 	  Enable Shell Commands for Counter and Timer
 
+config COUNTER_64BITS
+	bool
+	help
+	  This defines the counter ticks values (counter_ticks_t) to
```

For consistency with other hidden symbols/options descriptions:
```suggestion
	  Set by counter drivers to define counter ticks values (counter_ticks_t) to